src/multibody/geometry.hpp

```diff
@@ -220,20 +232,35 @@ namespace se3
     std::vector <DistanceResult> distance_results;
     
     ///
+    /// \brief Defines what information should be computed by collision test.
+    ///
+    fcl::CollisionRequest collisionRequest;
```

Comment:
Should this not be a vector rather than a unique object? It makes the API slightly more complex, but consistant with the two other vectors containing FCL objects. Maybe such a possibility would never be used...